ui/cypress/e2e/collectors.test.ts

```diff
@@ -165,5 +170,99 @@ describe('Collectors', () => {
       cy.getByTestID('resource-card').should('contain', secondTelegraf)
       cy.getByTestID('resource-card').should('not.contain', thirdTelegraf)
     })
+
+    it('can sort telegraf configs by name', () => {
+      const telegrafs = ['b', 'a', 'c']
+      const [firstTelegraf, secondTelegraf, thirdTelegraf] = telegrafs
+      const description = 'Config Description'
+
+      cy.get('@org').then(({id}: Organization) => {
+        cy.createTelegraf(firstTelegraf, description, id, bucket)
+        cy.createTelegraf(secondTelegraf, description, id, bucket)
+        cy.createTelegraf(thirdTelegraf, description, id, bucket)
+      })
+
+      cy.reload()
+
+      // test the order
+      // cy.getByTestID('resource-card').should('have.length', 3)
+      cy.getByTestID('collector-card--name')
+        .not('a')
+        .should('have.length', 3)
+
+      // test to see if telegrafs are initially sorted by name
+      telegrafs.sort()
+
+      cy.getByTestID('collector-card--name')
+        .each((val, index) => {
+          expect(val.text()).to.equal(telegrafs[index])
+        })
+        .then(() => {
+          cy.getByTestID('resource-list--sorter')
+            .contains('Name')
+            .click()
+          telegrafs.reverse()
+        })
+        .then(() => {
+          cy.getByTestID('collector-card--name').each((val, index) => {
+            expect(val.text()).to.equal(telegrafs[index])
+          })
+        })
```

Comment:
Couldn't find a working solution with `cypress` other than to chain statements. Would love to hear about any other implementations to guarantee serialization of tests

Reply:
chaining statements in cypress is totally fine.